lib/neutron

```diff
@@ -359,7 +359,11 @@ function install_neutron_new {
 
     # L3 service requires radvd
     if is_service_enabled neutron-l3; then
-        install_package radvd
+	if is_clearlinux; then
```

Comment:
Do the same thing here: ! is_clearlinux, adding openstack-common to files/rpms-clear/general covers all of these attempted installations automatically.